README.md

```diff
@@ -99,50 +99,76 @@ curl -u elastic http://127.0.0.1:9200/_cat/health
 Enter host password for user 'elastic':
 1472225929 15:38:49 docker-cluster green 2 2 4 2 0 0 0 0 - 100.0%
 ```
+## Configuration options
 
-### Logging
+The image offers several methods for [configuring Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/5.0/important-settings.html), presented below. The conventional approach is to provide a customized `elasticsearch.yml` file, but it's also possible to use environment variables to set options.
 
-Elasticsearch logs go to the console.
+### Bind-mounted configuration
+
+Create your own `custom_elasticsearch.yml` and override the default shipped with the image with the parameter:
+
+``` shell
+-v full_path_to/custom_elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml
+```
 
-### Notes for production use and defaults
+### Customized image
 
-1. Defining [elasticsearch parameters](https://www.elastic.co/guide/en/elasticsearch/reference/current/setup-configuration.html#settings) can be done with one of the following methods:
+In some environments, it may make more sense to prepare a custom image containing your configuration. A `Dockerfile` to achieve this may be as simple as:
 
-  - Create your own `custom_elasticsearch.yml` and override the default shipped with the image using `-v custom_elasticsearch.yml:/usr/share/elasticsearch/config/elasticsearch.yml`
-  - Present the parameters via docker environment variables.
-    For example to define the cluster name with docker run you'd need to pass `-e "cluster.name=mynewclustername"`. Double quotes are required.
+``` dockerfile
+FROM docker.elastic.co/elasticsearch/elasticsearch:5.0.0-rc1
+ADD elasticsearch.yml /usr/share/elasticsearch/config/
+```
+
+You could then build and try the image with something like:
+
+``` shell
+docker build --tag=elasticsearch-custom .
+docker run -ti -v /usr/share/elasticsearch/data -it elasticsearch-custom
```

Comment:
This is a good way to specify a named volume that gets assigned a random name. Will fix redundant `-ti`